### DISCUSSION TITLE
osu!lazer freezes randomly on MacOs

### DISCUSSION THREAD

**Discussion by un-juala** [ppy/osu#25507] [Q&A]
### Type

Crash to desktop

### Bug description

Not on startup but randomly during the start of some beatmaps, will freeze and become unresponsive. The Mac would be frozen on the screen and nothing could be done but force restart. 

### Version

osu! 2023.1114.1-lazer

### Logs

Sorry, no logs

**Comment by peppy** [comment]
Sounds like an OS or hardware failure. An app like osu! should not be able to take your system down.

Need logs and system specs.
